source/howtos/vpn_config.rst

```diff
@@ -150,8 +159,6 @@ Troubleshooting
 The version of OpenVPN we are running is incompatible with OpenVPN client v2.4+ on OS X.
 If you are running OS X, please use OpenVPN v2.3 to connect to the VPN.
 
-Disabling LZO compression may help on older OpenVPN instances.
```

Comment:
I think this was added on purpose in #37 . We should probably add this to the docs.

Reply:
Ah, gotcha. I was just assuming this was outdated. I'll just close this, then.